src/main/java/az/edu/turing/Main.java

```diff
@@ -4,7 +4,7 @@
 import az.edu.turing.controller.FamilyController;
 import az.edu.turing.dao.FamilyDao;
 import az.edu.turing.dao.impl.CollectionFamilyDao;
-import az.edu.turing.entities.*;
+import az.edu.turing.entity.*;
```

Comment:
avoid this type of import

Reply:
There is not other way. Beacuse we use all package fields.

Reply:
pls look
`[How do I stop wildcard Java package imports in IntelliJ](https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij)`